docs/cpp/config-clang-mac.md

```diff
@@ -102,20 +102,26 @@ In the `helloworld.cpp` file, hover over `vector` or `string` to see type inform
 
 You can press the `kbstyle(Tab)` key to insert the selected member. Then, when you add the opening parenthesis, you'll see information about arguments that the function requires.
 
-## Build helloworld.cpp
+## Run helloworld.cpp
+Remember, the C++ extension uses the C++ compiler you have installed on your machine to build your program. Make sure you have a C++ compiler installed before attempting to run and debug `helloworld.cpp` in VS Code.
 
-Next, you'll create a `tasks.json` file to tell VS Code how to build (compile) the program. This task will invoke the Clang C++ compiler to create an executable file from the source code.
+1. Open `helloworld.cpp` so that it is the active file.
+2. Press the play button in the top right corner of the editor.
 
-It's important to have `helloworld.cpp` open in the editor because the next step uses the active file in the editor as context to create the build task in the next step.
+   ![Screenshot of helloworld.cpp and play button](images/playbutton/run-play-button.png)
 
-From the main menu, choose **Terminal** > **Configure Default Build Task**.
-A dropdown will appear listing various predefined build tasks for the compilers that VS Code found on your machine. Choose **C/C++ clang++ build active file** to build the file that is currently displayed (active) in the editor.
+3. Choose **C/C++: clang++ build active file** from the list of detected compilers on your system (you'll only be asked to choose a compiler the first time you run `helloworld.cpp`).
```

Comment:
Please add this explanation (@sinemakinci1  feel free to re-phrase):

3. Choose **C/C++: clang++ build and debug active file** from the list of detected compilers on your system.
you'll only be asked to choose a compiler the first time you run `helloworld.cpp`. This compiler will be set as the "default" compiler in "tasks.json" file. In case you need to change the default compiler, you can run "Tasks: Configure default build task". Alternatively you can modify the "tasks.json" file and remove the default by replacing this:
```
            "group": {
                "kind": "build",
                "isDefault": true
            },
```
with this:
```
            "group": "build",
```

Reply:
I think you have to also replace "Tasks: Configure default build task" with something like `kb(workbench.action.tasks.)`

Reply:
as a side note: it is **C/C++: clang++ build and debug active file**, not **C/C++: clang++ build active file**